cdn_lambda/functions/origin_response/origin_response.py

```diff
@@ -0,0 +1,29 @@
+import logging
+import re
+
+LOG = logging.getLogger("origin-response")
+
+
+def lambda_handler(event, context):
+    # pylint: disable=unused-argument
+
+    request = event["Records"][0]["cf"]["request"]
+    original_uri = request["headers"]["original-uri"][0]["value"]
+
+    response = event["Records"][0]["cf"]["response"]
+
+    max_age_pattern_whitelist = [
+        ".+/PULP_MANIFEST",
+        ".+/listing",
+        ".+/repodata/repomd.xml",
+        ".+/ostree/repo/refs/heads/.+/.+",
```

Comment:
Those last two .+ should probably be [^/]+, right?  Else it actually matches any number of / >= 1, but if that's acceptable then we may as well just have written .*.

Reply:
regex was updated in latest commit, but I'm still not sure if it's correct